### ISSUE TITLE
[FR] Include an option for random delegation

### ISSUE THREAD

**Issue by onyxstakepool** [input-output-hk/daedalus#2166]
**Proposed feature:**
Include an additional delegation option in Daedalus to one or more randomly selected pools weighted by the non-myopic ranking.

**Rationale:**
Especially larger holders of ADA could use this option to distribute their holdings over many pools without the manual pool selection and curation effort. Daedalus would in the simplest case re-evaluate the random delegation every epoch and distribute the stake in chunks to many pools according to the non-myopic ranking.

**Benefit:**
The approach should accelerate the convergence of the stake distribution towards a uniform distribution over the k best pools measured by the non-myopic ranking. At the same time technically well run pools will benefit and lower quality pools will be sorted out more quickly.

**Pool operator dilemma:**
Currently pool operators face the challenge of attracting delegators on social media or by other means. With very low pool operating margins marketing efforts are difficult to fund. Usually technically well-versed operators are not marketing experts. Ultimately what the Cardano network needs are technical experts to run the distributed infrastructure. Thus only technical expertise should be incentivized by the system.

**Comment by latheesan-k** [comment]
As a small stake pool operator myself (ADABros) - I resonate with your thoughts; but isn't the staking portfolio idea Charles mentioned a better approach to give small stake pools to part of bigger picture?

**Comment by onyxstakepool** [comment]
> As a small stake pool operator myself (ADABros) - I resonate with your thoughts; but isn't the staking portfolio idea Charles mentioned a better approach to give small stake pools to part of bigger picture?
> 
> With portfolios - staking will become goals driven with actual real world benefits and even small guys like me can participate.

Portfolios are still marketing driven. I think the marketing effort is a distraction from the core task of the pool operator to run the network.

**Comment by latheesan-k** [comment]
> > As a small stake pool operator myself (ADABros) - I resonate with your thoughts; but isn't the staking portfolio idea Charles mentioned a better approach to give small stake pools to part of bigger picture?
> > With portfolios - staking will become goals driven with actual real world benefits and even small guys like me can participate.
> 
> Portfolios are still marketing driven. I think the marketing effort is a distraction from the core task of the pool operator to run the network.

That is true, I work for a digital market company yet I have no idea how best to advertise my pool to get delegation.

**Comment by disassembler** [comment]
Closing as part of issue backlog cleanup — issues older than 90 days are being retired to give the new team a clean slate. Please open a new issue if this is still relevant.
